### DISCUSSION TITLE
Leased/Rented Networks

### DISCUSSION THREAD

**Discussion by NMerrick58** [CMSgov/price-transparency-guide#376] [Q&A] [answered]
Are leased/rented networks expected to be including in the Machine Readable Files for In-Network/Allowed Amount? There is not much mentioned within the rule that points to this being required, only mentioned upon comments. This information would call for consistent aggregation of information coming from any network that is rented by the Issuer, and extends the complexity of the file for those Issuers completing in-house. If this is an expectation, how are other Issuers were handling the aggregation of these networks into their internal data? Thanks in advance!

**Comment by Devyandu** [comment, marked as answer]
The machine readable files should have provider contracted prices for all procedures for all plans.
 Some regional carriers might have leased networks for say, national coverage. Just because the regional carrier has leased the network from a national plan does not change the fact that those provider contracts (in the national network) are still valid contracts for the regional carrier's plan.
So, yes. the provider contracts for the leased network should be reported.

The table of contents/index file concept was created to make the above scenario easy to implement.
Continuing with the same example, the regional plan would publish the index file with all the plans they sell on their website.
For each plan in the index file, they would have two in-network files. One that the regional plan produces and the other from the national network. The national network file could very well be a URL pointing to the file residing on the national network's website.

Hope this helps

PS: I am not with CMS
